app/views/layouts/_edit_store.html.erb

```diff
@@ -2,6 +2,7 @@
     <div class="form-group">
```

Comment:
I like that this is in a partial, but it seems like we should have a single partial for both the new and edit forms rather than one that's just for the edit form. It's a bit of a pain because of the customization of the form_for for new vs edit, but you can do it using local variables sent in from the ' render layouts' line. I'm happy to help with this if it's not something you've seen before.